ansible/roles/captain/vars/main.yml

```diff
@@ -1,5 +1,9 @@
 deploy_env: captain
 # supervisor_task_conf: "{{ service_home }}/supervisor"
+#
+# Defined in config directory:
+# softlayer_vpn_username: ...
+# softlayer_vpn_password: ...
```

Comment:
Can I be included in this (offline) discussion when it happens? Also, not sure if this has already been deployed. If it has should the credentials be removed until we have made a decision?